CRITICAL - 2023-04-05 13:07:40 --> Class "Locale" not found
in SYSTEMPATH/CodeIgniter.php on line 194.
 1 ROOTPATH/spark(83): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-04-05 14:16:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:16:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:16:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:16:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:16:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:16:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:17:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:17:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:17:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:17:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:17:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:17:07 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
ERROR - 2023-04-05 14:50:30 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' in /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 /home/flac/Documents/apteka/back/app/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
#6 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(934): App\Controllers\Categories->index()
#7 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 /home/flac/Documents/apteka/back/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 /home/flac/Documents/apteka/back/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2023-04-05 14:50:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost'
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 SYSTEMPATH/Database/BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH/Controllers/Categories.php(25): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\Categories->index()
 7 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
CRITICAL - 2023-04-05 16:17:19 --> PhpOffice\PhpSpreadsheet\Writer\Xlsx::__construct(): Argument #1 ($spreadsheet) must be of type PhpOffice\PhpSpreadsheet\Spreadsheet, PhpOffice\PhpSpreadsheet\Worksheet\Worksheet given, called in /home/flac/Documents/apteka/back/app/Controllers/SubCategory.php on line 131
in VENDORPATH/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Writer/Xlsx.php on line 189.
 1 APPPATH/Controllers/SubCategory.php(131): PhpOffice\PhpSpreadsheet\Writer\Xlsx->__construct()
 2 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\SubCategory->generateExcelFile()
 3 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
CRITICAL - 2023-04-05 16:17:26 --> PhpOffice\PhpSpreadsheet\Writer\Xlsx::__construct(): Argument #1 ($spreadsheet) must be of type PhpOffice\PhpSpreadsheet\Spreadsheet, PhpOffice\PhpSpreadsheet\Worksheet\Worksheet given, called in /home/flac/Documents/apteka/back/app/Controllers/SubCategory.php on line 131
in VENDORPATH/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Writer/Xlsx.php on line 189.
 1 APPPATH/Controllers/SubCategory.php(131): PhpOffice\PhpSpreadsheet\Writer\Xlsx->__construct()
 2 SYSTEMPATH/CodeIgniter.php(934): App\Controllers\SubCategory->generateExcelFile()
 3 SYSTEMPATH/CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Commands/Server/rewrite.php(47): require_once('FCPATH/index.php')
